src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/__tests__/create_index_pattern.js

```diff
@@ -1,139 +0,0 @@
-import angular from 'angular';
```

Comment:
I removed this file because I think these types of assertions are better handled by a combination of functional tests and very narrowly-defined unit tests. I'll create unit tests for the new directives I've created.